tests/unit/date/core.js

```diff
@@ -1,19 +1,21 @@
 define( [
 	"jquery",
-	"globalize",
+	"./helper",
 	"date"
-], function( $, Globalize ) {
+], function( $, testHelper ) {
 
 module( "date: core" );
 
+var attributes = testHelper.getAttributes( "en" );
+
 test( "Instantiation", function() {
 	expect( 2 );
-	ok( new $.date() instanceof $.date, "constructor function" );
-	ok( $.date() instanceof $.date, "instantiation without new" );
+	ok( new $.ui.calendarDate( null, attributes ) instanceof $.ui.calendarDate, "constructor function" );
```

Comment:
I still think we should move it to `ui/`. The other "utilities" in there are also of limited use, like `ui/focusable.js`, which is only used by dialog.